src/components/[guild]/RoleCard/components/Reward.tsx

```diff
@@ -191,6 +191,10 @@ export type RewardIconProps = {
 const MotionImg = motion(Img)
 const MotionCircle = motion(Circle)
 
+const RewardIconFallbacks = {
+  discord: "platforms/discord.png",
```

Comment:
I think it would make sense to just use the icon from the reward config. So `rewards[PlatformType[guildPlatform.platformId]].imageUrl`. What do you think about this approach?